examples/SingleParticleCurrent/include/OneParticleSimulation.hpp

```diff
@@ -64,7 +64,7 @@ class OneParticleSimulation : public MySimulation
     {
     }
 
-    virtual uint32_t init()
+    virtual void init()
```

Comment:
I`am not sure if you can change the return type to void here (See [SO](http://stackoverflow.com/questions/4665117/c-virtual-function-return-type) post.

What about `return MySimulation::init()` ?